SunEngine.DataSeed/MainSeeder.cs

```diff
@@ -33,7 +40,24 @@ public MainSeeder(string configDirPath = "Config")
             providerName = dataBaseConfiguration["Linq2dbProvider"];
             connectionString = dataBaseConfiguration["ConnectionString"];
         }
-
+        
+        public bool СheckConnection()
+        {
+            try
+            {
+                DataBaseConnection db = new DataBaseConnection(providerName, connectionString);
```

Comment:
In this situation, it would be more appropriate to use `var` instead of full name of the type.